### PR TITLE
feat: add request offline floating license

### PR DESCRIPTION
**Pull request by muneebkq**
The `RequestOfflineFloatingLicense` function sends a request to lease the license for offline usage. The function takes a `leaseDuration` parameter, which specifies the duration of the lease in seconds. The function returns various status codes indicating the success or failure of the request. This feature allows users to use the license offline when an internet connection is not available.

### CODE REVIEW

File: lexfloatclient.go [hunk 1]
```diff
@@ -322,6 +322,22 @@ func HasFloatingLicense() int {
 	return int(status)
 }
 
+/*
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
+
+    RETURN CODES: LF_OK, LF_FAIL, LF_E_PRODUCT_ID, LF_E_LICENSE_EXISTS, LF_E_HOST_URL,
```
Comment (ahmad-kemsan) [inline lexfloatclient.go:352]: @muneebkq Parameter is missing in the docs. Also doc for this function is not complete.

File: lexfloatclient/LexFloatClient.h [hunk 2]
```diff
@@ -269,6 +269,18 @@ LEXFLOATCLIENT_API int LF_CC DropFloatingLicense();
 */
 LEXFLOATCLIENT_API int LF_CC HasFloatingLicense();
 
+/*
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
+
```
Comment (ahmad-kemsan) [inline lexfloatclient/LexFloatClient.h:281]: @muneebkq Parameter is missing in the docs. Also doc for this function is not complete.

**Review by ahmad-kemsan — CHANGES_REQUESTED** [review]
Please check the comments

File: lexfloatclient.go (hunk 1 above)
Comment (muneebkq) [reply lexfloatclient.go:352]: added now

File: lexfloatclient/LexFloatClient.h (hunk 2 above)
Comment (muneebkq) [reply lexfloatclient/LexFloatClient.h:281]: added

**Review by ahmad-kemsan — APPROVED** [review]

**Review by ahmad-kemsan — APPROVED** [review]

**Pull request merged**